pkg/outputter.go

```diff
@@ -215,9 +222,9 @@ func parseConfigTemplates(ctx context.Context, c *config.Config, iface *Interfac
 		InterfaceDir:            filepath.Dir(iface.FileName),
 		InterfaceDirRelative:    interfaceDirRelative,
 		InterfaceName:           iface.Name,
-		InterfaceNameCamel:      strcase.ToCamel(iface.Name),
-		InterfaceNameLowerCamel: strcase.ToLowerCamel(iface.Name),
-		InterfaceNameSnake:      strcase.ToSnake(iface.Name),
+		InterfaceNameCamel:      xstrings.ToCamelCase(iface.Name),
+		InterfaceNameLowerCamel: xstrings.FirstRuneToLower(xstrings.ToCamelCase(iface.Name)),
+		InterfaceNameSnake:      xstrings.ToSnakeCase(iface.Name),
```

Comment:
Does this have any behavior differences or are they strictly equivalent? I'm inclined to keep the old package populating these variables and just remove them in v3.

Reply:
as far as I see, they behave absolutely the same.
But since they are deprecated I guess it's okay to keep code unchanged.

Reply:
Done